app/src/helpers/helpers.ts

```diff
@@ -185,3 +185,8 @@ export function removeUserAgentSpecifics(
     .replace(`Electron/${process.versions.electron} `, '')
     .replace(`${appName}/${appVersion} `, ' ');
 }
+
+// Removes formatting from a text
+export function getPlainTextWithoutFormatting(text: string): string {
+  return text.trim().replace(/\n/g, ' ').replace(/\s+/g, ' ');
+}
```

Comment:
Uuuuuh, unless I'm missing something, this is not what's commonly understood as "copy as plain text", as implemented in popular text-processing programs. Copy as plain text means *stripping rich text* (a.k.a. HTML in a web context), so that e.g. ***bold italic*** text get pasted without the boldness/italicness.

If you rework this PR and are looking for to find a way to do this, it feels like there must be a way to do it with pure web/Electron APIs. Please don't use an extra dep (see [HACKING.md / 4. Avoid adding npm dependencies](https://github.com/nativefier/nativefier/blob/master/HACKING.md))

Reply:
@ronjouch I thought of the same thing, but when I looked at the the `clipboard.readText` function, it already strips out the HTML tags and markdown formatting. So there was no need to write a custom function to strip that out. All that was needed was to get rid of the newlines and extra spaces. When I tried it out, it did work as expected.

Reply:
@abhi12299 ooooooooh, I missed you call `clipboard.readText`, and the test you wrote test the stuff that happens *after* it; I was incorrectly understanding these tests as covering the full copying as plain text.

But also, this function name is very misleading. Let's rename it to what it does, not what the previously-called `clipboard.readText` does!